packages/components/pie-button/README.md

```diff
@@ -91,28 +91,22 @@ For example, to add a click handler in various templates:
 
 ```
 
-## TypeScript Enum Exports
+## TypeScript Type Exports
 
-For TypeScript projects, we export three enums related to button properties: `BUTTON_SIZE`, `BUTTON_TYPE`, and `BUTTON_VARIANT`. You can import and use these enums to set the corresponding property values for the `pie-button` component. This ensures better type safety and autocompletion in your project.
+For TypeScript projects, we export three types related to button properties: `ButtonSize`, `ButtonType`, and `ButtonVariant`. You can import and use these types to set the corresponding property values for the `pie-button` component. This ensures better type safety and autocompletion in your project.
 
 Here's an example of how to import and use the enums in a TypeScript project:
 
 ```typescript
-import { BUTTON_SIZE, BUTTON_TYPE, BUTTON_VARIANT } from '@justeattakeaway/pie-button';
-
-// Using the enums to set property values
-const myButtonSize = BUTTON_SIZE.LARGE;
-const myButtonType = BUTTON_TYPE.RESET;
-const myButtonVariant = BUTTON_VARIANT.SECONDARY;
+import type { ButtonSize, ButtonType, ButtonVariant } from '@justeattakeaway/pie-button';
```

Comment:
Super NIT: You've put `ButtonSize` (singular) in the README example, but used `ButtonSizes` (plural) in the actual imports in the example apps. Doesn't matter which one you go for (whichever makes most sense to you), but I'd make it consistent either way.